src/frontend/web_application/src/components/MessageList/components/Message/index.jsx

```diff
@@ -15,48 +34,119 @@ class Message extends Component {
 
   constructor(props) {
     super(props);
-    this.state = { isActive: false };
-    this.handleHover = this.handleHover.bind(this);
-    this.handleBlur = this.handleBlur.bind(this);
+    this.state = {
+      body: '',
+      isExpanded: true,
+      isTooLong: false,
+      isActive: false,
+    };
+    this.handleActiveClick = this.handleActiveClick.bind(this);
+    this.handleExpandClick = this.handleExpandClick.bind(this);
+  }
+
+  componentDidMount() {
+    this.setBodyHeight();
+  }
+
+  setBodyHeight() {
```

Comment:
I first tried to calculated height of the body div, and then, if it was > a certain height, collapse the div. For that I used `ref=` on `.m-message__body` BUT I add issues and that's why I used `body.length` instead of `.m-message__body`'s height to compare expanded/collapsed states.